test/GitHub.InlineReviews.UnitTests/ViewModels/InlineCommentThreadViewModelTests.cs

```diff
@@ -15,7 +16,8 @@ public class InlineCommentThreadViewModelTests
         public void CreatesComments()
         {
             var target = new InlineCommentThreadViewModel(
-                CreateSession(),
+                Substitute.For<ICommentService>()
+                , CreateSession(),
```

Comment:
Nit: this comment is in a weird position.

Reply:
comma you mean 😼